string obj_id
---
geometry_msgs/Pose2D nav_goal